src/todo-list/components/FinishedList.js

```diff
@@ -0,0 +1,77 @@
+import React, {useEffect} from 'react';
+import styled from 'styled-components';
+import useTodoList from '../hooks/useTodoList';
+import TodoItem from './TodoItem';
+
+function FinishedList() {
+
+    const { finishedTodos } = useTodoList();
+
+    useEffect(() => {
+      console.log('==================================')
+      console.log(' ')
+      console.log(' ')
+      console.log(' ')
+        console.log('finished: ', finishedTodos);
+        console.log(' ')
+        console.log(' ')
+        console.log(' ')
+        console.log('==================================')
+    }, [finishedTodos])
+
+    const renderFinishedTodos = () => {
+      return (
+          <StyledUl>
+            {finishedTodos.map(todo => 
+            <TodoItem todo={{id: todo.todoID, todoContent: todo.todoContent, waiting: todo.waiting}}  />)}
+          </StyledUl>
+      )
+    }
+
+    return (
+        <>
+            <StyledTitleContainer>
+                <StyledP>완료</StyledP>
+                <StyledP>{finishedTodos.length}</StyledP>
+            </StyledTitleContainer>
+            
+            <StyledSection>
+              {renderFinishedTodos()}
+            </StyledSection>
+        </>
+    )
+}
+
+const StyledTitleContainer = styled.div`
```

Comment:
```suggestion
const TitleContainer = styled.div`
```

개인적으로 `Styled`라는 접두어는 빼는게 좋다고 생각해요. 컴포넌트의 기능을 설명하는데 있어 어떠한 도움도 주지 않고, 쓸데없이 이름만 길어지게 된다고 봅니다.

Reply:
앗 넵 듣고보니 그렇네요 ㅎ...